jicofo/Dockerfile

```diff
@@ -12,6 +12,22 @@ RUN apt-dpkg-wrap apt-get update && \
     apt-dpkg-wrap apt-get install -y jicofo && \
     apt-cleanup
 
-COPY rootfs/ /
+COPY --chown=noroot:noroot rootfs/ /
+
+RUN mkdir -p /config
```

Comment:
Do we really need to try and create /config?

Reply:
Some of these mkdir statements used to be in the init script. considering the limited permissions of noroot, these directories need to be created with noroot as owner before that script in order to get over permission errors while running the container.

Reply:
Then let's put that in the base image please.